src/librustdoc/core.rs

```diff
@@ -424,62 +423,36 @@ crate fn run_global_ctxt(
         );
 
         if name == "no_default_passes" {
-            msg.help("you may want to use `#![doc(document_private_items)]`");
+            msg.help("`#![doc(no_default_passes)]` no longer functions; you may want to use `#![doc(document_private_items)]`");
+        } else if name.starts_with("passes") {
+            msg.help("`#![doc(passes = \"...\")]` no longer functions; you may want to use `#![doc(document_private_items)]`");
```

Comment:
Ugh, not super happy that this can be configured both through attributes and a CLI flags ... but that's a fight for another day.